src/Module.php

```diff
@@ -108,4 +118,9 @@ public function onAuthenticationPost(MvcAuthEvent $e)
 
         $this->container->setService('api-identity', $e->getIdentity());
     }
+
+    public function getMvcRouteListener()
```

Comment:
Never mind, I *do* see it in the unit tests. That said, I think there may be a better way to do that test; I'll see if I can come up with something.